src/main/java/io/github/nucleuspowered/nucleus/argumentparsers/ImprovedCatalogTypeArgument.java

```diff
@@ -43,13 +43,11 @@ public void parse(CommandSource source, CommandArgs args, CommandContext context
         } catch (ArgumentParseException e) {
             try {
                 if (!arg.contains(":")) {
-                    args.next();
```

Comment:
I need to do this differently, I just realised that this will break non-flag arguments.

I might as well just rewrite this argument and stop trying to be clever.